pennylane/qnode.py

```diff
@@ -182,6 +187,20 @@ def __init__(
         self.dtype = np.float64
         self.max_expansion = max_expansion
 
+    def __str__(self):
+        """String representation."""
+        return f'<QNode: wires={self.device.num_wires}, device="{self.device.short_name}", interface="{self.interface}", diff_method="{self.diff_method}">'
+
+    def __repr__(self):
+        """String representation."""
+        detail = "<QNode: wires={}, device='{}', interface='{}', diff_method='{}'>"
+        return detail.format(
+            self.device.num_wires,
+            self.device.short_name,
+            self.interface,
+            self.diff_method,
+        )

```

Comment:
It looks like `__str__` and `__repr__` currently return the same thing?

If so, best to simply define `__repr__` and remove `__str__` for now :)

Reply:
Hi! Would you recommend adding some other information for `__str__`. At this point, I don't know what could be added to it to make it different than `__repr__`.

Reply:
@paniash it's a good question, I'm also unsure what extra information would go in `__str__` 🤔 This potentially needs to be considered more, so I suggest removing `__str__` for now.